### PR TITLE
releases page re-design v2

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added (if applicable)
- [ ] permission has been obtained to add new logo (if applicable)
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

closes: https://github.com/AdoptOpenJDK/openjdk-website/issues/431

### CODE REVIEW

**Comment by jerboaa** [comment]
Have additional choice fields been considered? `Operating System` and `Architecture` come to mind. This would avoid the long list/scrolling.

**Comment by gdams** [comment]
yup that would deffo make things easier

File: src/handlebars/releases.handlebars [hunk 1]
```diff
@@ -37,23 +37,48 @@
...
+      <div class="filter-divs">
+        <div class="filter-os filter-div">
+          <span><strong>Operating System:</strong></span>
+          <select id="os-filter" class="filter" onchange="filterOS()">
+            <option value="Any">-Any-</option>
+            <option value="Linux">Linux</option>
+            <option value="macOS">macOS</option>
+            <option value="AIX">AIX</option>
+            <option value="Windows">Windows</option>
+            <option value="Solaris">Solaris</option>
```
Comment (jerboaa) [inline src/handlebars/releases.handlebars]: I'd have thought the OS list would be dynamically populated with all available OSes for which there is a release.

File: src/handlebars/releases.handlebars [hunk 2]
```diff
@@ -37,23 +37,48 @@
...
+          </select>
+        </div>
+
+        <div class="filter-arch filter-div">
+          <span><strong>Architecture:</strong></span>
+          <select id="arch-filter" class="filter" onchange="filterArch()">
+            <option value="Any">-Any-</option>
+            <option value="x64">x64</option>
+            <option value="x32">x32</option>
+            <option value="s390x">s390x</option>
```
Comment (jerboaa) [inline src/handlebars/releases.handlebars]: Same here. This gets you into a state where when "Linux" is selected the architecture list shows this static list which is not very intuitive. At least it confused me. I'd have expected for it to refresh and only show a list of arches for the corresponding OS selection.

File: src/handlebars/releases.handlebars (hunk 1 above)
Comment (gdams) [reply src/handlebars/releases.handlebars]: yeah I'll get that implemented today

**Comment by gdams** [comment]
@jerboaa updated to auto-populate the filters. On staging too

**Comment by karianna** [comment]
@gdams needs a rebase

**Comment by jerboaa** [comment]
@gdams This is nice! +1

**Pull request merged**